packages/jupyterlab-lsp/package.json

```
@@ -65,6 +65,7 @@
   },
   "devDependencies": {
     "@babel/preset-env": "^7.4.3",
+    "@jupyterlab-classic/application": "^0.1.1",
```

To grab the latest changes:

```suggestion
    "@jupyterlab-classic/application": "^0.1.2",
```

---

To be followed by a `yarn.lock` update.

---

Ah thanks, good catch :+1: 

It should be available now.